ajax/libs/react-semantify/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "react-semantify",
+  "filename": "react-semantify.min.js",
+  "description": "Integrate Semantic-ui with react components.",
+  "version": "0.4.1",
+  "keywords": [
+    "react",
+    "semantic-ui",
+    "semantic ui",
+    "react-component"
+  ],
+  "contributor": {
+    "name": "Lee",
+    "email": "jessy1092@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jessy1092/react-semantify"
+  },
+  "homepage": "http://jessy1092.github.io/react-semantify",
+  "license": "MIT",
+  "dependencies": {
+    "classnames": "^2.2.0",
+    "react": "^0.14.2",
+    "react-dom": "^0.14.2"
+  },
+  "npmName": "qwerty-hancock",
+  "npmFileMap": [
+    {
+      "basePath": "dst",
```

Comment:
@LeaYeh 
I think the npmName is `react-semantify`. https://www.npmjs.com/package/react-semantify

BTW. Thanks for adding `react-semantify` into cdnjs. :+1: